files-override/ts/eslint.config.mjs

```diff
@@ -0,0 +1,146 @@
+import globals from 'globals';
+import js from '@eslint/js';
+
+import ts from 'typescript-eslint';
+
+import ember from 'eslint-plugin-ember';
+import emberRecommended from 'eslint-plugin-ember/configs/recommended';
+import gjsRecommended from 'eslint-plugin-ember/configs/recommended-gjs';
+import gtsRecommended from 'eslint-plugin-ember/configs/recommended-gts';
+
+import prettier from 'eslint-plugin-prettier/recommended';
+import qunit from 'eslint-plugin-qunit';
+import n from 'eslint-plugin-n';
+
+import emberParser from 'ember-eslint-parser';
+import babelParser from '@babel/eslint-parser';
+
+const parserOptions = {
+  esm: {
+    js: {
+      ecmaFeatures: { modules: true },
+      ecmaVersion: 'latest',
+    },
+    ts: {
+      projectService: true,
+      tsconfigRootDir: import.meta.dirname,
+    },
+  },
+};
+
+export default ts.config(
```

Comment:
this utility is neat -- it re-adds some basic support for `extends`
-- its kind of a less-powerful version of [`configFor` + `forFiles`](https://github.com/NullVoxPopuli/eslint-configs/blob/main/configs/ember.js#L13-L16)

Here is proof this config works:
![image](https://github.com/user-attachments/assets/f82a30b8-32c6-42e4-ac05-6524e930fe57)